### DISCUSSION TITLE
🤖 [INFO]: Minimum Hardware / Working Setups ⚡

### DISCUSSION THREAD

**Discussion by trholding** [xai-org/grok-1#183] [Q&A]
###  Run grok-1 with less than 🔲 420 G VRAM 

See: [llama2.cpp / grok-1 support](https://github.com/ggerganov/llama.cpp/pull/6204)
[@ibab_ml on X](https://twitter.com/ibab_ml/status/1771340692364943750
)

###### Mac
- Mac Studio with an M2 Ultra 
- Total RAM: 192GB (unified RAM)

https://github.com/ggerganov/llama.cpp/pull/6204#issuecomment-2016392553

###### AMD
- CPU: Threadripper 3955WX
- Total RAM: 256GB RAM
- TPS: 0.5 tokens per second

https://github.com/ggerganov/llama.cpp/pull/6204#issuecomment-2016472333

###### Intel + Nvidia
- GPU: 8 x A100 80G 
- Total VRAM: 640G 
- CPU: 2 x Xeon 8480+
- RAM: 1.5 TB

https://github.com/xai-org/grok-1/discussions/168#discussioncomment-8834090

###### AMD
- GPU: 8 x Instinct MI300X GPU 190G 
- Total VRAM: 1520G 

https://github.com/xai-org/grok-1/issues/130#issuecomment-2005770022

###### Other / Container / Cloud 
- GPU: 8 x A100 80G 
- Total VRAM: 640G 
- K8 cluster

https://github.com/xai-org/grok-1/issues/6#issuecomment-2007301554

See: 
https://github.com/xai-org/grok-1/issues/42
https://github.com/xai-org/grok-1/issues/130#issuecomment-2004399998
https://github.com/xai-org/grok-1/issues/172#issuecomment-2005593309

**Comment by AndreSlavescu** [comment]
Made a slight edit to my original comment to eliminate ambiguity and confusion. H100GB should be H100 80GB.

**Comment by killatapez** [comment]
And what exactly happens when you try to launch this on some average PC config, lets say 3060ti/16gb RAM?

**Reply by fredex2** [reply]
![image](https://github.com/xai-org/grok-1/assets/3999778/74175733-cace-4c27-a6a0-f291f68df503)

**Reply by AndreSlavescu** [reply]
it will just throw a CUDA OOM error because it cannot load that much data on 16GB of VRAM

**Comment by jihuishan** [comment]
Thanks for sharing! So is it possible to do a little bit modification to run on 2 nodes with 4 A100 80g on each node?

**Reply by waxiliwwww** [reply]
Hello, have you manage to do that?

**Reply by jihuishan** [reply]
NO....
